C/src/xplaneConnect.c

```diff
@@ -516,15 +516,15 @@ int getDREFs(XPCSocket sock, const char* drefs[], float* values[], unsigned char
 	int result = sendDREFRequest(sock, drefs, count);
 	if (result < 0)
 	{
-		// A error ocurred while sending.
+		// An error ocurred while sending.
```

Comment:
Jeez. Basic grammar errors that survived for 5 years. Embarrassing.

Reply:
LOL!  Yes, it happens.